views/js/runner/plugins/tools/answerElimination/eliminator.js

```diff
@@ -144,12 +158,31 @@ define([
                 })
                 // commands that controls the plugin
                 .on(actionPrefix + 'toggle', function () {
+                    var $choices = self.$choiceInteractions.find('.qti-choice');
                     if (isEnabled()) {
                         self.$choiceInteractions.toggleClass('eliminable');
                         if (self.$choiceInteractions.hasClass('eliminable')) {
                             self.button.turnOn();
+                            if(config.restoreEliminationsOnOpen) {
+                                $choices.each(function() {
+                                    var input = this.querySelector('.real-label input');
+                                    if(!!this.dataset.wasEliminated) {
+                                        delete this.dataset.wasEliminated;
```

Comment:
Is the `delete` really needed? This can lead to memory leak. Prefer to use `null`.